src/checkoutservice/main.go

```diff
@@ -154,7 +131,6 @@ func initStackDriverTracing() {
 }
 
 func initTracing() {
-	initJaegerTracing()
```

Comment:
nit: this function should probably be removed if it contains a single line now

Reply:
done and done